### ISSUE TITLE
dap-ui-locals window keeps collapsing after every 'next'

### ISSUE THREAD

**Issue by obelhe** [emacs-lsp/dap-mode#811]
Hey there,

Trying to use dap mode with golang, I'm hitting a wall where everytime I go to the next statement the locals window collapses, so I have to navigate to that buffer and expand it

![Screenshot 2024-10-09 at 19 20 18](https://github.com/user-attachments/assets/24c2cb30-eaa7-4e52-aa4d-dd34dbb0dd2c)

I tried creating my own function that expands the locals window after going to the next statement. But that didn't work

```lisp
(defun next-expand ()
  (interactive)
  (dap--step "next" (dap--cur-session))

  (lsp-treemacs-render (dap-ui-locals-get-data) " Locals " dap-ui-locals-expand-depth dap-ui--locals-buffer)
  )
```
not sure if this is a dap thing or lsp-treemacs thing?

**Comment by edhilgendorf** [comment]
I'm having the same issue, @obelhe  did you figure something out?

**Comment by obelhe** [comment]
@edhilgendorf nope, i just gave up after trying multiple things that didn't work out. Went back to the good ol `fmt.Println`

**Comment by edhilgendorf** [comment]
@obelhe I tried the following but I get an error "No current debug session", do you know if there's another way to call these one after another?

```
(defun my-next-locals ()
  "Debug 'next' followed by expanding ui locals"
  (interactive)
  (dap-next) 
  (dap-ui-locals))
```

**Comment by edhilgendorf** [comment]
@yyoncho this seems to be broken again: https://github.com/emacs-lsp/dap-mode/issues/317

**Comment by nmaupu** [comment]
Same here, desperately trying to make it work correctly as well...

**Comment by elvinzeng** [comment]
I also encountered this problem, is there a solution?
